stdlib/2/email/charset.pyi

```diff
@@ -2,6 +2,10 @@ def add_charset(charset, header_enc=..., body_enc=..., output_charset=...) -> No
 def add_alias(alias, canonical) -> None: ...
 def add_codec(charset, codecname) -> None: ...
 
+QP: int       # undocumented
+BASE64: int   # undocumented
+SHORTEST: int # undocumented
```

Comment:
flake8 is failing, because we use PEP 8 comment style, i.e. exactly two spaces before the comment.

Reply:
Sorry. Fixed.